sample/src/main/java/kin/sdk/migration/sample/ChooseNetworkActivity.java

```diff
@@ -52,12 +53,28 @@ private void initWidgets() {
 
     private void createKinClient(KinClientSampleApplication.NetWorkType netWorkType) {
         KinClientSampleApplication application = (KinClientSampleApplication) getApplication();
-        IKinClient kinClient = application.createKinClient(netWorkType,"test");
-        if (kinClient.hasAccount()) {
-            startActivity(WalletActivity.getIntent(this));
-        } else {
-            startActivity(CreateWalletActivity.getIntent(this));
-        }
+        application.createKinClient(netWorkType, "test", new MigrationManagerListener() {
+
+            @Override
+            public void onMigrationStart() {
+                // TODO: 02/01/2019 add some progress bar
```

Comment:
what about all those TODOs?

Reply:
Will implement it later when we will finish with the important stuff